### DISCUSSION TITLE
BT-501-Organization-Company

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#658] [Q&A] [answered]
Hi devs, 

This field belongs under the node "ND-CompanyLegalEntity" and this node has only one field "BT-501-Organization-Company"

Field itself is not repeatable. Which make sense because its organization identifier which is quite unique. 
But node is repeatable. So practically its possible to have multiple organization identifiers. 

Is it correct or is it some mistake ? 

regards

**Comment by YvesJo** [comment, marked as answer]
Hi,
A given organization may indeed have various identifiers, also BT-501 is repeatable in the Regulation Annex; this repeatable aspect is covered in the implementation using a repeatable parent node.
Please for further non-technical questions, please get in touch with the helpdesk directly: ted@publications.europa.eu
